src/managers/TileManager.ts

```diff
@@ -44,46 +46,47 @@ class TileManager {
                 const dto: Nullable<TileDTO> =
                     dtos.find((dto: TileDTO) => dto.id === tile.id) || null;
 
-                let state: TileState = this.identifyTiles(
+                let state: TileState = this.determineState(
                     dto,
-                    session.turnPlayer!,
+                    session.turnPlayer,
                     i,
                 );
                 tile.apply(state, dto);
             }
-            this.getTilesInHand(); //TODO where to save/use
             this.onSync.emit();
         });
     }
 
-    private identifyTiles(
+    private determineState(
         dto: Nullable<TileDTO>,
-        turnPlayer: int,
-        i: int,
+        turnPlayer: Nullable<int>,
+        index: int,
     ): TileState {
+        const InitialAmount: int = this.getInitialAmount(
```

Comment:
camelCase

Reply:
i suggest moving this var outside the function and import it as a parameter so that it gets done only once and not for each tile